templates/repo/issue/list.tmpl

```diff
@@ -29,9 +29,7 @@
 		<div id="issue-filters" class="ui stackable grid">
 			<div class="six wide column">
 				{{if $.CanWriteIssuesOrPulls}}
-					<div class="ui checkbox issue-checkbox-all gt-vm">
-						<input type="checkbox" title="{{.locale.Tr "repo.issues.action_check_all"}}">
-					</div>
+					<input type="checkbox" autocomplete="off" class="issue-checkbox-all gt-vm gt-mr-4" title="{{.locale.Tr "repo.issues.action_check_all"}}">
```

Comment:
Does checkbox need `autocomplete="off"`?

Reply:
And, `title` vs tippy tooltip?

(and  below)

Reply:
> Does checkbox need `autocomplete="off"`?

I guess see https://github.com/go-gitea/gitea/pull/23596#discussion_r1148405657?

Reply:
I don't think it's the same thing. `autocomplete="off"` is designed for helping users fill forms, but not for playing with "state caching", see my updated comment.

Reply:
`autocomplete="off"` works and eliminates a batch of hacky JS. Also, GitHub does the same.

Reply:
It looks unclear to me. I think it really needs some document about: 

* Why there must be a "autocomplete=off" ? What it is used for?
* Is it related to Cache-Control? Does `no-store` resolve the problem?
* Does it affect all browsers? Or only some browsers?
* When the developers should (must) use this attribute? And when not?
* Could it be supported by framework level? eg: auto-add the attribute?

Reply:
I can add a comment, but to my knowledge this a kind of feature specific to Firefox.

> And, title vs tippy tooltip?

I'd question whether to just remove the title as it servers no real purpose and just echoes the issue title and I don't like these `«` characters in use there.

Reply:
But I guess you can't remove the `title` due to #22593

Reply:
1. I move the `title` on individual items to `aria-label`, I think it's a good compromise to keep it on the "check all" box.
2. I don't feel like explaining the checkbox autocomplete=off. Above stackoverflow link is the first google result for "checkbox autocomplete off" so any curious developer will find and understand it. We shouldn't need to litter our codebase with comments that explain easily googleable stuff.